CHANGELOG/CHANGELOG-v0.4.md

```diff
@@ -0,0 +1,23 @@
+# Changelog since v0.3.0
+All notable changes to this project will be documented in this file.
+
+## v0.4.0
+
+### New Features 
+* Support managedClusterAddon registration and maintain the lease of managedClusterAddon.
+
+### Added
+* Upgrade CRD to v1 and k8s lib to v0.21.0-rc.0.
```

Comment:
- Upgrade k8s api lib to v0.21.1
- Upgrade k8s client-go lib to v0.21.1
- Upgrade OCM CRDs to v1

Reply:
Could we put these to changes section?